src/app/features/ledger/components/ledger-inline-warnings.tsx

```diff
@@ -7,11 +7,44 @@ import { Capitalize } from '@app/ui/utils/capitalize';
 
 import { isStacksLedgerAppClosed } from '../utils/stacks-ledger-utils';
 
-interface CommonLedgerInlineWarningsProps {
+interface RequiresChainProp {
   chain: SupportedBlockchains;
+}
+
+interface CommonLedgerInlineWarningsProps extends RequiresChainProp {
   latestDeviceResponse: any | null;
   outdatedLedgerAppWarning?: boolean;
 }
+
+function OutdatedLedgerAppWarning({ chain }: RequiresChainProp) {
+  return (
+    <WarningLabel fontSize="14px" textAlign="left">
```

Comment:
You shouldn't need to set the `fontSize` here, it's set as `caption.02` in `WarningLabel`

Reply:
You mean in the text style? What if we wanted to centre align a caption?

Reply:
I don't mean in the textStyle, I should be clearer. 

I mean that we shouldn't have to pass `fontSize` as a prop as as in `WarningLabel` it's already using `caption.02` for children. I'm not sure our root font size, I guess it's 16? So `14px` = `0.875rem`.

`caption.02` size is `0.8125rem` which is actually `13px` so not the same as passed here but it probably should be the same as the others. 

![Screenshot 2023-11-28 at 09 47 41](https://github.com/leather-wallet/extension/assets/2938440/e0f264a2-c6ab-43e8-ab46-6d20d8344134)

It also has an icon there so we should probably always just `text-align='left'` in there (if its even needed now) then overwrite with centre when we do need it

Reply:
I think we can use the default warning label styles, will update